paseos/utils/link_budget_calc.py

```diff
@@ -0,0 +1,108 @@
+import os
+
+import numpy as np
+import pykep as pk
+from skyfield.api import load
+
+from .sky_field_sky_coordinate import SkyfieldSkyCoordinate
+from ..actors.ground_station_actor import GroundstationActor
+from ..actors.spacecraft_actor import SpacecraftActor
+
+_SKYFIELD_EARTH_PATH = os.path.join(
+    os.path.dirname(__file__) + "/../resources/", "de421.bsp"
+)
+# Skyfield Earth, in the future we may not always want to load this.
+_SKYFIELD_EARTH = load(_SKYFIELD_EARTH_PATH)["earth"]
```

Comment:
I believe this is already loaded in another file with the same name so this may create some weird interactions? (or at least load it twice). We might want to move it to the constants.py or similar?

Reply:
Moved it to constants.py